README.md

```diff
@@ -92,6 +92,10 @@ Axe can be built using your local language. To do so, a localization file must b
 
 `grunt build --lang=nl`
 
+or equivalently:
+
+`npm build -- --lang=nl`
```

Comment:
This seems to fail for me, but adding `run` seems to fix it.

```suggestion
`npm run build -- --lang=nl`
```

Reply:
Better yet, we should probably not promote the use of the `grunt` step and just use the `npm run build` process for this.

Reply:
"There should be one-- and preferably only one --obvious way to do it."